stream-api/src/main/kotlin/fund/cyber/markets/api/trades/TradesBroadcastersIndex.kt

```diff
@@ -12,9 +12,30 @@ class TradesBroadcastersIndex : TradesChannelsIndexUpdateListener {
         index.getOrPut(pair, { HashMap() }).put(exchange, broadcaster)
     }
 
-    fun broadcastersFor(pairs: List<TokensPair>): Collection<TokensPairTradesBroadcaster> {
+    fun broadcastersForPairs (pairs: List<TokensPair>): Collection<TokensPairTradesBroadcaster> {
```

Comment:
better to have single method, that take two lists and separate filter logic here, rather that duplicate it on all clients